### PR TITLE
Enable all managers

### PR DESCRIPTION
**Pull request by hopkiw**
Enable all managers appropriate to each OS
Correct a few small bugs

### CODE REVIEW

File: google_guest_agent/main.go [hunk 1]
```diff
@@ -115,11 +115,12 @@ func runUpdate() {
 	}
 
 	var wg sync.WaitGroup
-	var mgrs []manager
-	if runtime.GOOS == "windows" {
-		mgrs = []manager{newWsfcManager(), &addressMgr{}, &winAccountsMgr{}}
-	} else {
-		mgrs = []manager{&clockskewMgr{}}
+	mgrs := []manager{newWsfcManager(), &addressMgr{}}
```
Comment (adjackura) [inline google_guest_agent/main.go]: Do we want to enable wsfcManager?

Comment (hopkiw) [reply google_guest_agent/main.go]: i assumed so since it was before.....

Comment (adjackura) [reply google_guest_agent/main.go]: I meant for Linux

Comment (hopkiw) [reply google_guest_agent/main.go]: oh, duh. no we don't, moved into the windows block.

**Pull request merged**